uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/views/CheckoutViewPresenter.kt

```diff
@@ -168,15 +168,15 @@ internal class CheckoutViewPresenter(view: CheckoutViewContract.View,
         }
     }
 
-    override fun passBackPaymentIdentifiers(identifier: String, tripId: String?, passengerDetails: PassengerDetails?, comments: String) {
+    override fun passBackPaymentIdentifiers(identifier: String, tripId: String?, passengerDetails: PassengerDetails?, comments: String, flightInfo: String) {
         val passenger = if (KarhooUISDKConfigurationProvider.configuration
                         .authenticationMethod() is AuthenticationMethod.KarhooUser) parsePassengerDetails() else passengerDetails
 
         passenger?.let {
             val metadata = getBookingMetadataMap(identifier, tripId)
 
             tripsService.book(TripBooking(comments = comments,
-                                          flightNumber = flightDetails?.flightNumber,
+                                          flightNumber = flightInfo,
```

Comment:
if the flightInfo is empty, can we avoid sending it? 

if you pass `null` value to flightNumber, it will not be included in the request payload

```
val flight = if (flightInfo.isNotEmpty()) {
                flightInfo
            } else {
                null
            }
```